docs/components/password/index.html

```diff
@@ -0,0 +1,143 @@
+  elements/hx-password-control/index.html: <hx-password-control>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    Use {{page.title}} anytime a user must enter or update a password.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <header>
+      <h2 id="basic-password-input">Password</h2>
+      {# TODO: add section description #}
+    </header>
+     <p>Build out the password entry interaction including:
+        Meeting or not meeting password requirements.  Consider a popover on focus that interactively displays met criteria.
+        Showing and hiding password text.
+    </p>
+
+    <div class="example" id="vue-passwordInputDemo" v-cloak>
+      <header>
+        <form class="beta-hxForm">
+          <fieldset>
+            <legend class="beta-hxFieldName">Label Annotation</legend>
+            <hx-checkbox-control>
+              <input
+                id="chkHasAsterisk"
+                type="checkbox"
+                v-model="hasAsterisk"
+              />
+              <label for="chkHasAsterisk">
+                <hx-checkbox></hx-checkbox>
+                Required
+              </label>
+            </hx-checkbox-control>
+            <hx-checkbox-control>
+              <input
+                id="chkHasHelpText"
+                type="checkbox"
+                v-model="hasHelpText"
+              />
+              <label for="chkHasHelpText">
+                <hx-checkbox></hx-checkbox>
+                Help Text
+              </label>
+            </hx-checkbox-control>
+            <hx-checkbox-control>
+              <input
+                id="chkHasError"
+                type="checkbox"
+                v-model="hasError"
+              />
+              <label for="chkHasError">
+                <hx-checkbox></hx-checkbox>
+                Error Text
+              </label>
+            </hx-checkbox-control>
+          </fieldset>
+
+          <fieldset>
+            <legend class="beta-hxFieldName">Control Options</legend>
+            <hx-checkbox-control>
+              <input
+                id="chkDisabled"
+                type="checkbox"
+                v-model="isDisabled"
+              />
+              <label for="chkDisabled">
+                <hx-checkbox></hx-checkbox>
+                Disabled
+              </label>
+              <p>
+                (Invalid styling applied <i>after</i> user interaction.)
+              </p>
+            </hx-checkbox-control>
+
+            <hx-checkbox-control :class="{ hxHelpText: hasHelpText}" v-if="helpTextToDisplay">
+                <label> Help Text</label>
+              <input type="text" v-model="helpText">
+             </hx-checkbox-control>
+
+             <hx-checkbox-control :class="{hxErrorTextForInput: hasError}" v-if="errorTextToDisplay">
+                <label> Error Text</label>
+              <input type="text" v-model="errorText">
+             </hx-checkbox-control>
```

Comment:
Please always show these options.  No need to hide them.

Reply:
Changes have been done, please review.